resolwe_bio/docker_images/common/packages-debian.txt

```diff
@@ -17,3 +17,6 @@ libfindbin-libs-perl
 
 # Required for deeptools
 libpython3.6-dev
+
+# Required for MultiQC
+python3-setuptools
```

Comment:
@mzganec this was moved from `packages-debian-buildreq.txt` to here, since newer version of Multiqc requires  `python3-setuptools`